src/trio/_tests/test_exports.py

```diff
@@ -319,8 +319,11 @@ def lookup_symbol(symbol: str) -> dict[str, str]:
         if module_name == "trio.socket" and class_name in dir(stdlib_socket):
             continue
 
-        # ignore class that does dirty tricks
-        if class_ is trio.testing.RaisesGroup:
+        # Ignore classes that don't use attrs, they only define their members once
+        # __init__ is called (and reason they don't use attrs is because they're going
```

Comment:
Wouldn't classes that use `__slots__` define members before `__init__` as well? It's just that they wouldn't have any values.

Reply:
ye I suppose `__slots__` can also make this test work. Pytest seems to only use it sparingly though so don't think I will bother.